[Docs] field types fix typos (#178)
Sylius/Stack: docs/grid/field_types.md

</details>

In the `@Grid/Column/_prettyName.html.twig` template, you just need to
render the value. For example:

```twig
<strong>{{ data }}</strong>
```

> If you wish to render more complex grid fields, just redefine the path of
> the field to root – `path: .` in the yaml file and you can access all
> attributes of the object instance:

Reviewer: yaml mention can be skipped, I think it should be same for php config file as well.

```suggestion
If you wish to render more complex grid fields, just redefine the path of
the field to root – `path: .` and then you can access all
attributes of the object instance:
```

Author: Done :)
